### PR TITLE
Add common test usings to SonarLint.Test.props

### LINKED ISSUE #5396: Add common test usings to global usings
add these usings into the test directory.build.targets file like this:

https://github.com/SonarSource/sonar-dotnet/blob/3622c4a4126173fd74812d205e4e490fbddb548a/analyzers/tests/SonarAnalyzer.Test/SonarAnalyzer.Test.csproj#L41-L44

### PR DESCRIPTION
**Pull request by georgii-borovinskikh-sonarsource**
Fixes #5396

### CODE REVIEW

File: src/SonarLint.Test.props [hunk 1]
```diff
@@ -26,5 +26,12 @@
     <PackageReference Include="FluentAssertions" Version="5.9.0" />
     <PackageReference Include="FluentAssertions.Analyzers" Version="0.11.4" />
   </ItemGroup>
+  
+  <ItemGroup>
```
Comment (pavel-mikula-sonarsource) [inline src/SonarLint.Test.props:30]: For the entire project (production + test code), we have also this:
https://github.com/SonarSource/sonar-dotnet/blob/3622c4a4126173fd74812d205e4e490fbddb548a/analyzers/Analyzers.targets#L16-L24

Can you add it too, maybe in a separate PR? All except the immutable collections seems relevant in SL project

**Review by pavel-mikula-sonarsource — APPROVED** [review]
LGTM

File: src/SonarLint.Test.props (hunk 1 above)
Comment (pavel-mikula-sonarsource) [reply src/SonarLint.Test.props:30]: The least used is `System.Diagnostics`. The purpose of this one is to be able easily do `Debugger.Break()` or `Debugger.Launch` anywhere

Comment (georgii-borovinskikh-sonarsource) [reply src/SonarLint.Test.props:30]: https://github.com/SonarSource/sonarlint-visualstudio/issues/5426

**Pull request merged**